extension/landing-zone/variables.tf

```diff
@@ -0,0 +1,328 @@
+variable "ibmcloud_api_key" {
+  description = "The API key that's associated with the account to provision resources to"
+  type        = string
+  sensitive   = true
+}
+
+variable "resource_group" {
+  type        = string
+  description = "An existing resource group name to use for this example."
+}
+
+variable "region" {
+  description = "The region of the existing LZ VPC."
+  type        = string
+}
+
+variable "prefix" {
+  description = "The prefix that you would like to append to VSI, Block Storage, Security Group, Floating IP and Load Balancer."
+  type        = string
+  default     = "slz-vsi"
+}
+
+variable "vpc_id" {
+  description = "The ID of the VPC where the VSI is created."
+  type        = string
+}
+
+variable "ssh_keys" {
+  description = "SSH keys to use to provision a VSI. Must be an RSA key with a key size of either 2048 bits or 4096 bits (recommended). If `public_key` is not provided, the named key will be looked up from data. See https://cloud.ibm.com/docs/vpc?topic=vpc-ssh-keys."
```

Comment:
So if I understand correct, if `name` and `public_key` are both passed into the variable, it will add the public key to the account. However if just `name` is passed, it will lookup an existing key in the region provided with that name and use that?

@SirSpidey I might pull you in for re-wording here to make this clearer

Reply:
Yes, that's right.